Extension/src/LanguageServer/ui_new.ts

```diff
@@ -296,7 +296,7 @@ export class NewUI implements UI {
         if (settings.intelliSenseEngine === "disabled") {
             this.intelliSenseStatusItem.text = this.missingIntelliSenseText;
             this.intelliSenseStatusItem.command = {
-                command: "C_Cpp.OpenCompilerQuickpick",
+                command: "C_Cpp.SelectDefaultCompiler",
```

Comment:
It seems this whole block could be deleted because there won't be a language status if the intelliSenseEngine is "disabled"?

Reply:
I'm not sure what the original thinking was about this code.  Based on the message, I think the check it wrong, and the intention was to direct the user to `SelectDefaultCompiler` if IntelliSense get not get properly configured (not because IntelliSense was disabled).